### PR TITLE
[dvsim] Fix remaining comments  from #5876

### PR DESCRIPTION
**Pull request by sriyerg**
Signed-off-by: Srikrishna Iyer <sriyer@google.com>

### CODE REVIEW

File: util/dvsim/Deploy.py [hunk 1]
```diff
@@ -203,9 +203,9 @@ def _process_exports(self):
     def _construct_cmd(self):
         """Construct the command that will eventually be launched."""
 
-        args = ["make", "-f", self.flow_makefile, self.target]
+        cmd = "make -f {} {}".format(self.flow_makefile, self.target)
```
Comment (rswarbrick) [inline util/dvsim/Deploy.py:206]: I don't think this is the right thing to do. If you're struggling to debug things (as described in https://github.com/lowRISC/opentitan/pull/5876#discussion_r606430536) and can't spend the time to fix it properly, maybe it's best to ask someone for help? "I changed this and something I don't understand broke so I'm leaving it be" is not a great way to maintain software, in my opinion.

Comment (sriyerg) [reply util/dvsim/Deploy.py:206]: There is some degree of urgency to get this in. Filing an issue so that we fix it later.

Comment (sriyerg) [reply util/dvsim/Deploy.py:206]: Filed issue #5945 to ensure this gets dealt with soon.

Comment (dsandrs) [reply util/dvsim/Deploy.py:206]: @rswarbrick @sriyerg I agree with Sri - we need to get this in quickly, or we risk getting cutoff from our tools.

**Review by dsandrs — APPROVED** [review]

**Comment by imphil** [comment]
Calling other tools is a key functionality of dvsim, and as such, we need to get this functionality right and fully understand our own code.

This PR is clearly a workaround for a problem that is not understood. I can see the desire to paper over a problem during time pressure, and as such am approving this PR.

But even in such a case (and even more so!) be explicit and clear about it in the pull request and commit message. 

Before merging, please rewrite the commit message to clearly indicate that this is a temporary workaround for a yet undiagnosed issue. Link to the follow-up issue you filed. In the follow-up issue (https://github.com/lowRISC/opentitan/issues/5945), explain how to reproduce the issue that led to the workaround you're applying in this PR, the behavior you see, etc.

**Review by imphil — APPROVED** [review]

**Pull request merged**